retired/modules/dmrpp_module/ngap_container/unit-tests/bes.conf.in

```diff
@@ -1,22 +1,22 @@
 
 BES.LogName = ./bes.log
-BES.Catalog.catalog.RootDirectory=@abs_top_srcdir@/modules/ngap_module
-BES.Catalog.catalog.TypeMatch=dmrpp:.*\.(dmrpp)$;
+# BES.Catalog.catalog.RootDirectory=@abs_top_srcdir@/modules/ngap_module
```

Comment:
Did you intend to comment all this configuration stuff out? Just checking...

Reply:
Yes. Those configuration files are not used anymore. I meant to move the .in versions to retired.